bridge/github/import.go

```diff
@@ -23,61 +23,46 @@ const (
 // githubImporter implement the Importer interface
 type githubImporter struct {
 	conf core.Configuration
+
+	// iterator
+	iterator *iterator
+
+	// number of imported issues
+	importedIssues int
+
+	// number of imported identities
+	importedIdentities int
 }
 
 func (gi *githubImporter) Init(conf core.Configuration) error {
 	gi.conf = conf
 	return nil
 }
 
+func (gi *githubImporter) Reset() {
+	gi.importedIssues = 0
```

Comment:
This doesn't fully do what the function's name imply: you still have the iterator laying around with the current iteration state.

Pro-tip: write the documentation (especially for public functions). This will force you to phrase clearly what the function do or should do, and you will see what is missing or wrong, if any.

Reply:
That's true, i changed the name to `ResetCounters`. Also added some comments to all the non commented public functions.

Reply:
Actually i don't know if we really need the `ResetCounters` function here, the only use case i can imagine would be calling `ImportAll` multiple times (with different repositories ?)

Reply:
`ResetCounters` doesn't make sense if you can't also reset the whole iterator and start from zero. But it's easier and cheap to just create a new one, so why bother ?

Reply:
Agreed, It would be better to just recreate a new one